builder.js

```diff
@@ -80,6 +80,7 @@ module.exports = {
   },
   optimization: {
     splitChunks: {
+      name: 'vendors',
```

Comment:
why is that needed?

Reply:
I was tired of changing head.html every time. Maybe there are some implications I'm not aware of and it should be reverted.